src/main/java/org/zalando/nakadi/validation/EventTypeOptionsValidator.java

```diff
@@ -28,11 +27,11 @@ public void validate(final Object target, final Errors errors) {
     }
 
     private void checkRetentionTime(final Errors errors, final EventTypeOptions options) {
-        if (Objects.nonNull(options) && Objects.nonNull(options.getRetentionTime())) {
-            final Long retentionTime = options.getRetentionTime();
-            if (retentionTime > maxTopicRetentionMs) {
+        final Long retentionTime = options.getRetentionTime();
+        if (retentionTime != null) {
+            if (retentionTime.longValue() > maxTopicRetentionMs) {
```

Comment:
1. It is a little bit faster in some cases when unboxing.
2. maxTopicRetentionMs is long and for me it is more clearer to compare primitive types rather than reference and primitive.
Maybe it is just my old habit from previous projects with the older java versions :)